### PR TITLE
iwctl: add page and French translation

### PR DESCRIPTION
**Pull request by D34DPlayer**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).

### CODE REVIEW

**Review by owenvoke — APPROVED** [review]
I can't verify the French translation, but LGTM!

File: pages/linux/iwctl.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+- List your wifi stations:
+
+`iwctl station list`
+
+- Start looking for networks with a station:
+
+`iwctl station {{station}} scan`
+
+- List the networks found by a station:
```
Comment (navarroaxel) [inline pages/linux/iwctl.md:21]:
```suggestion
- Display the networks found by a station:
```

File: pages/linux/iwctl.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+- Start the interactive mode, in this mode you can enter the commands directly, with autocompletion:
+
+`iwctl`
+
+- Call general help:
+
+`iwctl --help`
+
+- List your wifi stations:
```
Comment (navarroaxel) [inline pages/linux/iwctl.md:13]:
```suggestion
- Display your wifi stations:
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]
Hi @D34DPlayer, thanks for your submission. I left you a few changes.

File: pages.fr/linux/iwctl.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+- Lancer le mode interactif, dans ce mode vous pouvez entrer les commandes directement, avec de l'autocompletion:
+
+`iwctl`
+
+- Avoir l'aide générale:
+
+`iwctl --help`
+
+- Lister vos stations wifi:
```
Comment (D34DPlayer) [inline pages.fr/linux/iwctl.md:13]:
```suggestion
- Afficher vos stations wifi:
```

File: pages.fr/linux/iwctl.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
+`iwctl station list`
+
+- Lancer la recherche de réseaux avec une station:
+
+`iwctl station {{station}} scan`
+
+- Lister les réseaux trouvés par une station:
```
Comment (D34DPlayer) [inline pages.fr/linux/iwctl.md:21]:
```suggestion
- Afficher les réseaux trouvés par une station:
```

**Review by D34DPlayer — COMMENTED** [review]
Changed the translation to fit the suggested changes in the English file.

File: pages/linux/iwctl.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
+# iwctl
+
+> A command line tool for controlling the iwd network supplicant.
```
Comment (navarroaxel) [inline pages/linux/iwctl.md:3]: I wonder if the link to the official wiki page is useful.
```suggestion
> A command line tool for controlling the iwd network supplicant.
> More information: <https://iwd.wiki.kernel.org/gettingstarted>.
```

**Review by navarroaxel — APPROVED** [review]
Thanks @D34DPlayer!

File: pages/linux/iwctl.md (hunk 5 above)
Comment (owenvoke) [reply pages/linux/iwctl.md:3]: I did think about that, but it doesn't look that useful. 🤔

Comment (navarroaxel) [reply pages/linux/iwctl.md:3]: at least it's better to know where start reading.

**Review by schneiderl — APPROVED** [review]
Don't know the language but looks good to me.

File: pages/linux/iwctl.md (hunk 5 above)
Comment (owenvoke) [reply pages/linux/iwctl.md:3]: Fair point. 👍🏻

Comment (navarroaxel) [reply pages/linux/iwctl.md:3]: Plase @D34DPlayer: could you add the link in both pages so we can merge this? Thanks!

File: pages.fr/linux/iwctl.md [hunk 6]
```diff
@@ -0,0 +1,27 @@
+# iwctl
+
+> Un outil de ligne de commande pour gèrer iwd.
```
Comment (D34DPlayer) [inline pages.fr/linux/iwctl.md:3]:
```suggestion
> Un outil de ligne de commande pour gèrer iwd.
> Plus d'informations: <https://iwd.wiki.kernel.org/gettingstarted>.
```

**Pull request merged**